plugins/Diagnostics/lang/en.json

```diff
@@ -0,0 +1,8 @@
+{
+    "Diagnostics": {
+        "ConfigFileTitle": "Config file",
+        "ConfigFileIntroduction": "Here you can view the Piwik configuration. If you are running Piwik in a load balanced environment the page might be different depending from which server this page is loaded. Rows with a different background color are changed config values that are specified for example in the %s file.",
+        "HideUnchanged": "If you want to see only changed values you can %shide all unchanged values%s.",
```

Comment:
@sgiehl if we want to enforce this (a good idea!), ideally we would add an integration test that fails when two or more `%s` are found... and fails with a useful message of what to do.